### DISCUSSION TITLE
Avoid slide duplication when only one element change

### DISCUSSION THREAD

**Discussion by 95A31** [marp-team/marp#527] [Q&A] [answered]
Hello,
I have several slides describing a process, where the majority of the content is common, and only one image changes. I am wondering if there is a way to avoid duplicating the slides and keep everything clean and compact.

Now I have 

```markdown
---
marp: true
---

## Title
Long text 
Long text 
Long text 
<img src=image1.png">

---

## Title
Long text 
Long text 
Long text 
<img src=image2.png">
```

I would like something like

```markdown
---
marp: true
---

## Title
Long text 
Long text 
Long text 

<section data-marpit-alternatives">
    <img src=image1.png" data-marpit-alternative>
    <img src=image2.png" data-marpit-alternative>
</section>

---
```

Thank you

**Comment by yhatt** [comment, marked as answer]
The build/incremental slide support in Marp is intentionally minimal because that is against the common practice for making better presentations. So there is no specific syntax to avoid duplicated contents currently.

It is necessary to need to make [a new Marp (Marpit) plugin](https://marpit-api.marp.app/) to add syntax for generating incremental slides page with duplicated content.

**Reply by 95A31** [reply]
Thank you for the clarification!
